src/routes/notifications.routes.js

```diff
@@ -0,0 +1,19 @@
+import express from "express";
+import paths from "../constants/paths.const.js";
+import controllers from "../controllers/notifications.controllers.js";
+
+const router = express.Router();
+
+// GET: /api/v1/notifications/user/:user_id
+router.get(
+  `${paths.notifications.base}/user/:user_id`,
+  controllers.getNotificationsByUserId,
+);
+
+// GET: /api/v1/notifications/user/:user_id/:notification_id
+router.get(
+  `${paths.notifications.base}/user/:user_id/:notification_id`,
```

Comment:
`${paths.notifications.base}/:notification_id/user/:user_id`

this is a suggestion to change the endpoint path.

Reply:
Thanks, I changed as suggested